ReleaseNotes.md

```diff
@@ -1,3 +1,7 @@
+# 1.0.4
+
+- [Feature] Added support for using default credentials instead of an access token for authentication.

```

Comment:
I'll add this after the merge and bump it to 1.1.0 to reflect the magnitude of the changes here